src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -118,7 +118,7 @@ public void load(final File file, final Component ui) {
       }
       message = "Exception while parsing: " + file.getName() + " : "
           + (gameName.get() != null ? gameName.get() + " : " : "") + message;
-      System.out.println(message);
+      ClientLogger.logQuietly(e);
```

Comment:
When this error occurs, the message displayed to the user states, "Please see console for full error log," but there was nothing there other than the exception message already displayed in the UI.  So I changed this line to log the entire stack trace.